### PR TITLE
feat(python): add egg and wheel analyzer

### PR DESCRIPTION
**Pull request by ankk13**

### CODE REVIEW

File: analyzer/library/egg/egg.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	"github.com/aquasecurity/fanal/types"
+)
+
+func init() {
+	analyzer.RegisterAnalyzer(&eggLibraryAnalyzer{})
+}
+
+const version = 1
+
+var requiredFiles = []string{".egg-info", ".egg-info/PKG-INFO"}
```
Comment (knqyf263) [inline analyzer/library/egg/egg.go:22]: Could you consider running it on Windows? We haven't done that, but we hope to do so in the future.

**Review by knqyf263 — COMMENTED** [review]
Please add tests

File: types/const.go [hunk 2]
```diff
@@ -15,6 +15,8 @@ const (
 	Pip      = "pip"
 	Pipenv   = "pipenv"
 	Poetry   = "poetry"
+	Egg      = "egg"
+	Wheel    = "wheel"
```
Comment (knqyf263) [inline types/const.go:19]: I think we can aggregate them into `python`.

File: go.sum [hunk 3]
```diff
@@ -185,8 +185,10 @@ github.com/apparentlymart/go-textseg v1.0.0/go.mod h1:z96Txxhf3xSFMPmb5X/1W05FF/
 github.com/apparentlymart/go-textseg/v12 v12.0.0/go.mod h1:S/4uRK2UtaQttw1GenVJEynmyUenKwP++x/+DdGV/Ec=
 github.com/apparentlymart/go-textseg/v13 v13.0.0 h1:Y+KvPE1NYz0xl601PVImeQfFyEy6iT90AvPUL1NNfNw=
 github.com/apparentlymart/go-textseg/v13 v13.0.0/go.mod h1:ZK2fH7c4NqDTLtiYLvIkEghdlcqw7yxLeM89kiTRPUo=
-github.com/aquasecurity/go-dep-parser v0.0.0-20210802113332-f09967d68412 h1:l9f0keIaQpYo1cKfm3abwHY/skXXNPM65cu4I/I9TgI=
-github.com/aquasecurity/go-dep-parser v0.0.0-20210802113332-f09967d68412/go.mod h1:Cv/FOCXy6gwvDbz/KX48+y//SmbnKroFwW5hquXn5G4=
+github.com/aquasecurity/go-dep-parser v0.0.0-20210520015931-0dd56983cc62 h1:aahEMQZXrwhpCMlDgXi2d7jJVNDTpYGJOgLyNptGQoY=
+github.com/aquasecurity/go-dep-parser v0.0.0-20210520015931-0dd56983cc62/go.mod h1:Cv/FOCXy6gwvDbz/KX48+y//SmbnKroFwW5hquXn5G4=
+github.com/aquasecurity/go-dep-parser v0.0.0-20210802141821-9c3e354a42b8 h1:F6wC7YbnTlSMIJKRhoPsb3388hImxuxf6KEploWXrxs=
+github.com/aquasecurity/go-dep-parser v0.0.0-20210802141821-9c3e354a42b8/go.mod h1:Cv/FOCXy6gwvDbz/KX48+y//SmbnKroFwW5hquXn5G4=
```
Comment (knqyf263) [inline go.sum:191]: `go mod tidy`?

File: analyzer/const.go [hunk 4]
```diff
@@ -30,6 +30,9 @@ const (
 	TypePipenv   Type = "pipenv"
 	TypePip      Type = "pip"
 	TypePoetry   Type = "poetry"
+	TypeEgg      Type = "egg"
+	TypeWheel    Type = "wheel"
+
```
Comment (knqyf263) [inline analyzer/const.go:35]: nit: remove newline

**Review by knqyf263 — CHANGES_REQUESTED** [review]
Looks almost good

File: types/const.go (hunk 2 above)
Comment (ankk13) [reply types/const.go:19]: Yes, I too thought that. It its okay with you to merge them both, let me know where do you want to merge at fanal or at go-dep-parser.

**Review by knqyf263 — APPROVED** [review]

**Pull request merged**